### PR TITLE
Adding OCP tag in the RHEL-OSE Vagrantfile for CDK

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: components/rhel/rhel-ose/Vagrantfile [hunk 1]
```diff
@@ -5,6 +5,9 @@
 # This variable is ignored for Hyper-V provider.
 PUBLIC_ADDRESS="10.1.2.2"
 
+#Image tag for OCP
+IMAGE_TAG="v3.3.1.3"
```
Comment (LalatenduMohanty) [inline components/rhel/rhel-ose/Vagrantfile:9]: @praveenkumar , Should we comment the line and user can un-comment if he want?

Comment (LalatenduMohanty) [reply components/rhel/rhel-ose/Vagrantfile:9]: @praveenkumar we can not comment this as we are using the variable in the Vagrantfile.

**Comment by LalatenduMohanty** [comment]
Tested it and it works fine.

**Comment by praveenkumar** [comment]
LGTM 👍

**Pull request merged**